src/Form/RegistrationFormType.php

```diff
@@ -53,8 +54,12 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     new NotBlank([
                         'message' => 'Merci de renseigner un mot de passe.',
                     ]),
+                    new Regex([
+                        'pattern' => '/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,}$/',
```

Comment:
You should probably not forbid to to have non alphanumeric characters in a password